src/rgw/rgw_admin.cc

```diff
@@ -336,6 +336,7 @@ void usage()
   cout << "   --gen-access-key                  generate random access key (for S3)\n";
   cout << "   --gen-secret                      generate random secret key\n";
   cout << "   --key-type=<type>                 key type, options are: swift, s3\n";
+  cout << "   --key-active=<active>             activate or deactive a key\n";
```

Comment:
deactivate?  Or perhaps here we want "options are: active, inactive" ?

Reply:
thanks, i fixed the misspellings. i changed this part to `--key-active=<bool>` to clarify what kind of value it expects